wrappers/python/simtk/openmm/app/desmonddmsfile.py

```diff
@@ -400,21 +657,23 @@ def _addNonbondedForceToSystem(self, sys):
             else:
                 new_epsilon =  b_ij**2/(4*a_ij)
                 new_sigma = (a_ij / b_ij)**(1.0/6.0)
+                
             nb.addException(p0, p1, q_ij, new_sigma, new_epsilon, True)
-
+            
         n_total = self._conn.execute("""SELECT COUNT(*) FROM pair_12_6_es_term""").fetchone()
         n_in_exclusions = self._conn.execute("""SELECT COUNT(*)
         FROM exclusion INNER JOIN pair_12_6_es_term
         ON exclusion.p0==pair_12_6_es_term.p0 AND exclusion.p1==pair_12_6_es_term.p1""").fetchone()
         if not n_total == n_in_exclusions:
-            raise NotImplementedError('All pair_12_6_es_terms must have a corresponding exclusion')
-
-        return nb
+            print('Warning: not all pair_12_6_es_terms have a corresponding exclusion')
```

Comment:
Are there situations when this isn't an error?

Reply:
Right, sorry, we missed all of the unit testing things. We'll repropose once done.

I think we will use IMPACT to get reference energies rather than Desmond. I am actually not sure how to give .dms files to recent versions of Desmond. And in any case, the implementation of OPLS in Desmond was originally benchmarked against IMPACT ... 

Concerning: 

> Are there situations when this isn't an error?

None that I know of. Thank you for pointing that out. That check needs to be fixed as it does not account for reversed indexes in the two tables (0,1 vs. 1,0 for example). We meant to do it but it fell through the cracks. It will be fixed.